pytorch_lightning/loggers/file_logger.py

```diff
@@ -0,0 +1,180 @@
+"""
+File logger
+-----------
+"""
+import io
+import os
+import csv
+import torch
+
+from argparse import Namespace
+from typing import Optional, Dict, Any, Union
+
+
+from pytorch_lightning import _logger as log
+from pytorch_lightning.core.saving import save_hparams_to_yaml
+from pytorch_lightning.loggers.base import LightningLoggerBase
+from pytorch_lightning.utilities.distributed import rank_zero_only
+
+
+class ExperimentWriter(object):
+    NAME_HPARAMS_FILE = 'hparams.yaml'
+    NAME_METRICS_FILE = 'metrics.csv'
+
+    def __init__(self, log_dir):
```

Comment:
typing :)

Reply:
Hi, I'm not sure what you mean by "typing"

Reply:
Adding type signatures to arguments and return types. for your init it would be `def __init__(self, log_dir: Union[str, path lib.Path]) -> None:`